doc/articles/guides/uno-material-walkthrough.md

```diff
@@ -229,20 +225,30 @@ This guide will walk you through the necessary steps to setup and to use the [`U
         <Color x:Key="MaterialOnErrorColor">#000000</Color>
         <Color x:Key="MaterialOverlayColor">#51FFFFFF</Color>
 
-        <!-- ... --->
+        <!-- ... -->
     </ResourceDictionary>
 
-    <!-- ... --->
+    <!-- ... -->
     ```
     > You may also use this for picking colors: https://material.io/design/color/the-color-system.html#tools-for-picking-colors
-1. In `App.xaml.cs`, update the line that initializes the material library to include the new palette:
-    ```cs
-    protected override void OnLaunched(LaunchActivatedEventArgs e)
-    {
-         Uno.Material.Resources.Init(this, new ResourceDictionary { Source = new Uri("ms-appx:///Styles/Application/ColorPaletteOverride.xaml") });
-
-        // [existing code...]
-    }
+1. In `App.xaml`, update the line that initializes the `MaterialColors` to include the new palette override:
+    ```xml
+    <Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <!-- ... -->
+
+                <!-- Load Material Color Palette with ColorPaletteOverrideSource -->
+				<MaterialColors xmlns="using:Uno.Material"
+								ColorPaletteOverrideSource="ms-appx:///ColorPaletteOverride.xaml" />
```

Comment:
Shouldn't ColorPaletteOverride.xaml be a `<ResourceDictionary/>` inserted between `MaterialColors` and `MaterialResources`?

Reply:
Like how is it done for the UWP Fluent system?

Reply:
Nah, we need to do some hacky stuff with the override when initializing the colors, specifically for UWP. Each time a RD is initialized that includes the colors as a merged dictionary, it creates a brand new instance of the color RD and blow away the override unless we force the override to be added as a merged dictionary with each new instance

Reply:
That's weird (I'm not sure I understand fully here...) so overriding the fluent colors doesn't work?
The ones from here: https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-theme-resources#base

Reply:
I don't understand it either. But on Windows, all the styles are in one giant ResourceDictionary. The problem arises when you have something like `Button.xaml` and have it "import" another RD using `MergedDictionaries`. It will create a new instance of that imported RD